.github/20-feature-request.md

```diff
@@ -0,0 +1,27 @@
+---
+name: Feature Request
+about: Use this template for raising a feature request
+
+---
+
+<em>tag:feature_template</em>
+
+**Please provide the following information**
+
+**System information**
+- Tensorboard version (you are using) :
+- Are you willing to contribute it (Yes/No) :
+
+
+
+**Describe the feature and the current behavior/state.**
```

Comment:
To make it consistent with other files, please end it with ":" or change those files to end with the period.

Reply:
Done